rsmtool/utils.py

```diff
@@ -37,16 +37,24 @@
                   'PositiveLassoCV']
 
 SKLL_MODELS = ['AdaBoostRegressor',
+               'BayesianRidge',
                'DecisionTreeRegressor',
+               'DummyRegressor',
                'ElasticNet',
                'GradientBoostingRegressor',
+               'HuberRegressor',
                'KNeighborsRegressor',
+               'Lars',
                'Lasso',
+               'LinearRegression',
```

Comment:
If we expose SKLL Linear Regression, shall it be added to `skll_linear_models` above?

Reply:
Actually, it doesn't make any sense to expose this since we already have `LinearRegression` in our built in models list. I will remove this.